highlight.io/components/QuickstartContent/backend/java/shared-snippets.tsx

```diff
@@ -0,0 +1,74 @@
+import { siteUrl } from '../../../../utils/urls'
+import { QuickStartStep } from '../../QuickstartContent'
+
+export const installSdk: QuickStartStep = {
+	title: 'Install the Highlight Java SDK.',
+	content: 'Add Highlight to your maven pom file.',
+	code: {
+		text: `<dependency>
+	<groupId>io.highlight</groupId>
+	<artifactId>highlight-sdk</artifactId>
+	<version>latest</version>
+</dependency>`,
+		language: 'xml',
+	},
+}
+
+export const initializeSdk: QuickStartStep = {
+	title: 'Initialize the Highlight Java SDK.',
+	content: '`Highlight.init()` initializes the Highlight backend SDK.',
+	code: {
+		text: `HighlightOptions options = HighlightOptions.builder("YOUR_PROJECT_ID")
```

Comment:
could this be `<YOUR_PROJECT_ID>` so that our frontend correctly replaces this?
we should really make a constant for that string...

Reply:
ah, okay